dozer-sql/src/pipeline/expression/builder.rs

```diff
@@ -300,7 +303,13 @@ impl ExpressionBuilder {
                             fun: gft,
                             args: function_args.clone(),
                         }),
-                        Err(_err) => Err(InvalidNestedAggregationFunction(function_name)),
+                        Err(_e) => match ConditionalExpressionType::new(function_name.as_str()) {
```

Comment:
Is it possible to have a sequence of try and fail? rather than a nested one?

Reply:
got it fixed